yaml/com/ibm/Panel.interface.yaml

```diff
@@ -0,0 +1,85 @@
+description: >
+    Implements control panel activities.
+methods:
+    - name: Display
+      description: >
+          Commands the op-panel to display the given lines.
```

Comment:
Reword?: API to display data on LCD panel. 
Since there are two panels, its will be handy if we explicitly tell that.

Reply:
done